src/sortStudents.ts

```diff
@@ -1,16 +1,85 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
```

Comment:
I'll recomend gice value for enum
```suggestion
enum SortType {
  name = 'name ',
  surname = 'surname ',
  age= 'age',
  married= 'married',
  averageGrade= 'averageGrade',
}

```